docker/docker.settings.php

```diff
@@ -18,3 +18,12 @@
 $settings['container_yamls'][] = __DIR__ . '/services.yml';
 
 $settings['install_profile'] = 'minimal';
+
+// Development specific configuration
+// Load Drupal default development configuration
+require_once DRUPAL_ROOT . '/sites/example.settings.local.php';
```

Comment:
It seems wrong to load the example file? We should keep a settings.local.php file that we mount inside the `/sites/` directory or similar.